core/flyout_base.js

```diff
@@ -536,6 +536,22 @@ Blockly.Flyout.prototype.show = function(xmlList) {
         } else {
           gaps.push(default_gap);
         }
+      } else if ((tagName == 'BUTTON') &&
```

Comment:
Why is this not a nested if inside the tagName == BUTTON || tagName == LABEL check?

Reply:
I felt this way was clearer, because this part of the code will soon change and create a different type of button.